src/main/resources/application.conf

```diff
@@ -0,0 +1,15 @@
+{
+  AUTH_TOKENS  {
+    roomId1
+      {
+        "token": authToken1
+      }
+    roomId2
+      {
+        "token": authToken2
+      }
+      .
+      .
+      .
+  }
+}
```

Comment:
Still need to check if the editor settings add EOL newline char at the end of the file.  Sometimes it causes problems without EOL.  Or always append a blank line at the end?